src/strategies/spidStrategy.ts

```diff
@@ -129,6 +63,31 @@ const spidStrategy = (
     }
   };
 
+  const IDP_IDS: { [key: string]: string } = {
```

Comment:
some keys may not have associated elements:

```suggestion
  const IDP_IDS: { [key: string]: string | undefined } = {
```

Reply:
fixed on commit 5323876